catalogue/webapp/pages/images.tsx

```diff
@@ -168,13 +120,7 @@ const Images: NextPage<Props> = ({
             : []
         }
       >
-        <Space
-          v={{
-            size: 'l',
-            properties: ['padding-bottom'],
-          }}
-          className="row"
```

Comment:
Slowly removing this class from everywhere as it's not declared and is therefore not useful